backend/spec/controllers/spree/admin/users_controller_spec.rb

```diff
@@ -168,14 +167,14 @@ def user
     context "when the user cannot manage roles" do
       it "cannot set roles" do
         expect {
-          put :update, params: { id: user.id, user: { first_name: "Bob", spree_role_ids: [dummy_role.id] } }
+          put :update, params: { id: user.id, user: { name: "Bob Bloggs", spree_role_ids: [dummy_role.id] } }
         }.not_to change { user.reload.spree_roles.to_a }
       end
 
       it "cannot clear roles" do
         user.spree_roles << dummy_role
         expect {
-          put :update, params: { id: user.id, user: { first_name: "Bob" } }
+          put :update, params: { id: user.id, user: { name: "Bob Bloggs" } }
```

Comment:
I see this as a missing opportunity to add `Bob Marley` to the repository 😆

Reply:
Ahah true 😂 
I tried to use boring names already present in specs to minimize surprise

Reply:
That makes perfect sense, I was just kidding!